.yardopts

```diff
@@ -0,0 +1,2 @@
+app/models/mixins/supports_feature_mixin.rb
+app/models/manager_refresh/inventory_collection.rb
```

Comment:
> To add files to the documentation, simply add them to .yardopts

Is this actually what this does, excluding the rest of the files from the documentation? I thought this file specification was for adding files that wouldn't be used by default (`.rb` is a default so I would assume all the docs would be compiled as they currently are)

Also, if this really does just include the files above and nothing else....why? I see no reason you shouldn't just include everything, it's still useful to see a class interface and if anything motivates to start writing docs for things you look up but find there's no documentation written for.

Reply:
It will only produce output for the selected files, yes.
Why? Because I rather have few discoverable documentation than a huge ocean of just method names without doc value.

Go to http://www.rubydoc.info/github/manageiq/manageiq - browse a bit around. I guess you wont come back again, cause there is no real documentation in that. Compare that to http://www.rubydoc.info/github/rails/rails/

Reply:
I guess ¯\\\_(ツ)_/¯ I think it's still useful to be able to search classes/methods to see if they even exist. Rails isn't 100% fully documented either, it just has a lot more than us :)

Maybe we can leave this as you have it and it can be changed once we have more documentation at all.